### DISCUSSION TITLE
Spatial thermal comfort error

### DISCUSSION THREAD

**Discussion by sanwitiab** [UMEP-dev/UMEP#574] [Q&A]
Hi, I tried to do a post-processor with Spatial Thermal Comfort tools. However, after I set every input parameter and press run.
I got this error message.

241, in processAlgorithm
build = gdal_buildings.ReadAsArray().astype(float)
AttributeError: 'NoneType' object has no attribute 'ReadAsArray'

**Comment by nilswallenberg** [comment]
Hi,

You seem to be missing the buildings raster layer. When you run SOLWEIG, remember to tick "Save necessary raster(s) for the TreePlanter and Spatial TC tools."

Cheers,
Nils

**Reply by sanwitiab** [reply]
So, I already checked to "Save necessary raster(s)...". The buildings.tif file showed in result folder with the others file.
Anyway, when I look at the help information for these tools. I found a little bit difference with my interface. For me it does not have the input box for building and meteorological data.
![image](https://github.com/UMEP-dev/UMEP/assets/59221006/03be3507-ff15-4cf4-bc73-6e245bc3d278)

**Reply by biglimp** [reply]
Can you show the content of your folder where you have **Tmrt_2022_21_1500D.tif**, please.

**Reply by sanwitiab** [reply]
This is the content in the result folder.
![image](https://github.com/UMEP-dev/UMEP/assets/59221006/898f1d21-4ef8-460d-9a92-d711e93e819d)

**Reply by biglimp** [reply]
Hm, this is strange. @nilswallenberg do you have any suggestions? @sanwitiab , can you also zip and attach the data above (together with your windspeed result) here.

**Reply by sanwitiab** [reply]
For sure here is the files.
[sanwit_solweig_post.zip](https://github.com/UMEP-dev/UMEP/files/13456836/sanwit_solweig_post.zip)

**Reply by biglimp** [reply]
I cannot replicate your error. Its runs om my computer.
![image](https://github.com/UMEP-dev/UMEP/assets/11908356/bb5989eb-d752-4631-83dc-2771f557efaa)

**Reply by sanwitiab** [reply]
I got it . What should I do for fixed it?

**Reply by biglimp** [reply]
not sure. Try to locate your files at a different location on your computer, closer to your root. Try another computer. Do you have the latest version of UMEP for Processing?
